<launch>
 <!-- <node name="tf_odom_to_map" pkg="tf" type="static_transform_publisher" args= "0 0 0 0 0 0 camera_link  base_footprint 10"/> -->
  <node name="orb_slam2_rgbd" pkg="orb_slam2_ros"
      type="orb_slam2_ros_planning" output="screen">

       <remap from="/camera/rgb/image_raw" to="/camera/rgb/image_rect_color" />
       <remap from="/camera/depth_registered/image_raw" to="/camera/depth_registered/hw_registered/image_rect" />

       <param name="publish_pointcloud" type="bool" value="true" />
       <param name="publish_pose" type="bool" value="true" />
       <param name="localize_only" type="bool" value="true" />
       <param name="reset_map" type="bool" value="false" />

       <!-- static parameters -->
       <param name="load_map" type="bool" value="true" />
       <param name="map_file" type="string" value="/home/onepiece/Dobriq/tracking3.bin" />
       <param name="settings_file" type="string" value="$(find orb_slam2_ros)/orb_slam2/config/astra.yaml" />
       <param name="voc_file" type="string" value="$(find orb_slam2_ros)/orb_slam2/Vocabulary/ORBvoc.txt" />

       <param name="pointcloud_frame_id" type="string" value="map" />
       <param name="camera_frame_id" type="string" value="camera_link" />
       <param name="min_num_kf_in_map" type="int" value="5" />
  </node>
  <include file="$(find astra_camera)/launch/astrapro.launch"> </include>
</launch>
